### PR TITLE
Displaying error message when the project is private/invalid

### LINKED ISSUE #22: Display an error test when project does not exist
When adding a new project which does not exist. E.g. `invalid/invalid`, an error text should show `The 'invalid/invalid' project does not exist!`

### PR DESCRIPTION
**Pull request by shravan-balasubramanian**
Fixes #22

### CODE REVIEW

**Comment by shravan-balasubramanian** [comment]
I have used the default alert method of JS. If you want the alert to be customized, please mention it

File: src/components/Search Component/Search.js [hunk 1]
```diff
@@ -13,6 +18,10 @@ export default class Search extends React.Component {
   handleChange = (e) => {
     this.setState({input:e.target.value})
   }
+
+  alertUser = (message) => { 
+    alert(message);
```
Comment (haroon-sheikh) [inline src/components/Search Component/Search.js]: instead of an error, can we show the error below the search bar please

Comment (shravan-balasubramanian) [reply src/components/Search Component/Search.js]: Are there any other designs for it?

Comment (haroon-sheikh) [reply src/components/Search Component/Search.js]: Whatever you feel will make it easier. Go creative! ;)

**Comment by shravan-balasubramanian** [comment]
Sorry for the delay. Stuck up with work

**Comment by haroon-sheikh** [comment]
@shravan-balasubramanian Thank you for your contributions. Happy Hacktober! Feel free to come back and contribute to this and other projects.

**Pull request merged**